controllers/grid/queries/QueriesGridCellProvider.php

```diff
@@ -69,24 +71,24 @@ public function getTemplateVarsFromRowColumn($row, $column)
     {
         $element = $row->getData();
         $columnId = $column->getId();
-        assert($element instanceof \PKP\core\DataObject && !empty($columnId));
+        assert(($element instanceof DataObject || $element instanceof Model) && !empty($columnId));
         /** @var Query $element */
         $headNote = $element->getHeadNote();
-        $user = $headNote ? $headNote->getUser() : null;
-        $notes = $element->getReplies(null, NoteDAO::NOTE_ORDER_ID, \PKP\db\DAO::SORT_DIRECTION_DESC);
+        $user = $headNote?->getUser();
+        $notes = $element->getReplies(null, Note::NOTE_ORDER_ID, \PKP\db\DAO::SORT_DIRECTION_DESC);
         $context = Application::get()->getRequest()->getContext();
         $datetimeFormatShort = PKPString::convertStrftimeFormat($context->getLocalizedDateTimeFormatShort());
 
         switch ($columnId) {
             case 'replies':
                 return ['label' => max(0, $notes->count() - 1)];
             case 'from':
-                return ['label' => ($user ? $user->getUsername() : '&mdash;') . '<br />' . ($headNote ? date($datetimeFormatShort, strtotime($headNote->getDateCreated())) : '')];
+                return ['label' => ($user ? $user->getUsername() : '&mdash;') . '<br />' . ($headNote ? date($datetimeFormatShort, strtotime($headNote->dateCreated)) : '')];
```

Comment:
`$headNote->dateCreated` should be a `Carbon` object now, so `strtotime` is no longer appropriate here (and below).